src/transport_ofi.c

```diff
@@ -72,9 +72,9 @@ uint8_t**                       shmem_transport_ofi_target_heap_addrs;
 uint8_t**                       shmem_transport_ofi_target_data_addrs;
 #endif /* ENABLE_REMOTE_VIRTUAL_ADDRESSING */
 #endif /* ENABLE_MR_SCALABLE */
-uint64_t                        shmem_transport_ofi_pending_put_counter;
-uint64_t                        shmem_transport_ofi_pending_get_counter;
-uint64_t                        shmem_transport_ofi_pending_cq_count;
```

Comment:
OPA_int_t is 32-bit, and these counters will overflow if the size is reduced to 32-bits.  😞

Reply:
It reduces portability, but I think the best approach will be to rely on C11 atomics: http://en.cppreference.com/w/c/atomic

We can also write our own wrappers so that we can reduce the overhead in builds where the user disables threads.